### ISSUE TITLE
loss.backward is blocked

### ISSUE THREAD

**Issue by YaaoTu** [facebookresearch/XLM#45]
Hi,
Thanks a lot for the awesome project~
I append a MLP after XLM sentence embedding to build a QA model. But after run several step(single GPU, 21000 step, 8 batch size), it is blocked on loss.backward step, without any error message.  If run on 4 GPU, it will blocked sooner (like 4200 step, 4*8 batch size). Could you please give some hint how can I fix this bug?
Thanks a lot~

**Comment by glample** [comment]
Hi,

Sorry for the late reply. Can you copy the full traceback with the error?
